UIKitDZ/UIKitDZ/VC/PaymentViewController.swift

```diff
@@ -0,0 +1,187 @@
+        let label = UILabel(frame: CGRect(x: 30, y: 120, width: 200, height: 30))
+        label.text = "Ваш заказ:"
+        label.font = .systemFont(ofSize: 26)
+        return label
+    }()
+    var namePizzaLabel: UILabel {
+        let label = UILabel(frame: CGRect(x: 30, y: 160, width: 300, height: 40))
+        label.text = pizzaNameLabel?.capitalized
+        label.font = .boldSystemFont(ofSize: 30)
+        return label
+    }
+    let addExtrasLabel: UILabel = {
+        let label = UILabel(frame: CGRect(x: 30, y: 220, width: 300, height: 30))
+        label.text = "Допольнительно вы выбрали:"
+        label.font = .systemFont(ofSize: 22)
+        return label
+    }()
+    let oneExtrasIngredintLabel: UILabel = {
+        let label = UILabel(frame: CGRect(x: 30, y: 260, width: 300, height: 30))
+        label.text = ""
+        return label
+    }()
+    let twoExtrasIngredintLabel: UILabel = {
+        let label = UILabel(frame: CGRect(x: 30, y: 300, width: 300, height: 30))
+        label.text = ""
+        return label
+    }()
+    let threExtrasIngredintLabel: UILabel = {
+        let label = UILabel(frame: CGRect(x: 30, y: 340, width: 300, height: 30))
+        label.text = ""
+        return label
+    }()
+    let fourExtrasIngredintLabel: UILabel = {
+        let label = UILabel(frame: CGRect(x: 30, y: 380, width: 300, height: 30))
+        label.text = ""
+        return label
+    }()
+    let paymentLabel: UILabel = {
+        let label = UILabel(frame: CGRect(x: 50, y: 560, width: 300, height: 30))
+        label.text = "Выберите способ оплаты:"
+        label.font = .systemFont(ofSize: 22)
+        return label
+    }()
+    let cardPaymentLabel: UILabel = {
+        let label = UILabel(frame: CGRect(x: 50, y: 600, width: 200, height: 30))
+        label.text = "Картой"
+        label.font = .systemFont(ofSize: 26)
+        return label
+    }()
+    let cardPaymentSwitch: UISwitch = {
+        let cardSwitch = UISwitch(frame: CGRect(x: 300, y: 600, width: 0, height: 0))
+        return cardSwitch
+    }()
+    
+    let cashPaymentLabel: UILabel = {
+        let label = UILabel(frame: CGRect(x: 50, y: 640, width: 200, height: 30))
+        label.text = "Наличными"
+        label.font = .systemFont(ofSize: 26)
+        return label
+    }()
+    let cashPaymentSwitch: UISwitch = {
+        let cashSwitch = UISwitch(frame: CGRect(x: 300, y: 640, width: 0, height: 0))
+        return cashSwitch
+    }()
+    let payButton: UIButton = {
+        let button = UIButton(frame: CGRect(x: 45, y: 720, width: 300, height: 50))
+        button.setImage(UIImage(systemName: "applelogo"), for: .normal)
+        button.imageView?.tintColor = .white
+        button.setTitle(" Pay", for: .normal)
+        button.titleLabel?.font = .boldSystemFont(ofSize: 20)
+        button.backgroundColor = .black
+        button.layer.cornerRadius = 5
+        return button
+    }()
+    var pizzaNameLabel: String?
+    var isAddChessBool: Bool?
```

Comment:
```suggestion
    var isAddChess: Bool?
```
приставка  is уже понятно что бул